docs/api.md

```diff
@@ -3602,12 +3602,12 @@ const { chromium } = require('playwright');  // Or 'firefox' or 'webkit'.
 
 <!-- GEN:toc -->
 - [browserType.connect(options)](#browsertypeconnectoptions)
-- [browserType.downloadBrowserIfNeeded([progress])](#browsertypedownloadbrowserifneededprogress)
 - [browserType.executablePath()](#browsertypeexecutablepath)
 - [browserType.launch([options])](#browsertypelaunchoptions)
 - [browserType.launchPersistentContext(userDataDir, [options])](#browsertypelaunchpersistentcontextuserdatadir-options)
 - [browserType.launchServer([options])](#browsertypelaunchserveroptions)
 - [browserType.name()](#browsertypename)
+- [browserType.setExecutablePath(executablePath)](#browsertypesetexecutablepathexecutablepath)
```

Comment:
We discussed this offline, let's make it private for now, e.g. `_setExecutablePath`, or exposed similar to download utility.

Reply:
Removed.